packages/mdc-menu/simple/mdc-simple-menu.scss

```diff
@@ -53,50 +55,48 @@ $mdc-simple-menu-item-fade-duration: .3s;
     outline: none;
   }
 
+  &--animating-open {
+    display: inline-block;
+    overflow-y: hidden;
+    opacity: 0;
+    transform: scale(0.8);
+    transition: opacity $mdc-simple-menu-fade-duration linear,
+                transform $mdc-simple-menu-scale-duration $mdc-animation-deceleration-curve-timing-function;
+    > .mdc-simple-menu__items {
+      transform: scale(1.25);
+    }
+  }
+
   &--open {
     display: inline-block;
     transform: scale(1);
     opacity: 1;
+    > .mdc-simple-menu__items {
+      transform: scale(1);
+    }
   }
 
-  &--animating {
+  &--animating-close {
```

Comment:
See earlier comment RE renaming this to `--animating-closed`

Reply:
done